src/backend/views/layout.twig

```diff
@@ -7,7 +7,7 @@
     <meta property="og:title" content="{{ page.title | striptags }}" />
     <meta property="article:modified_time" content="{{ (page.body.time / 1000) | date("c") }}" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
-    <link rel="icon" type="image/png" href="/favicon.png">
+    <link rel="icon" type="image/png" href="/favicon">
```

Comment:
favicons can be on other formats, not pngs only

<img width="644" alt="image" src="https://user-images.githubusercontent.com/32655339/174228857-2497aa97-3439-450c-83a6-6842e2c00d7f.png">

I think we could handle these cases